pyomo/common/tests/test_fileutils.py

```diff
@@ -99,6 +100,19 @@ def test_this_file_dir(self):
         expected_path = os.path.join('pyomo','common','tests')
         self.assertTrue(_this_file_dir.endswith(expected_path))
 
+    def test_import_file(self):
+        import_ex = import_file(os.path.join(_this_file_dir, 'import_ex.py'))
```

Comment:
Out of curiosity, what happens if the file is `import ex.py` or `import-ex.py`?

Reply:
Tested it in ipython with a file `import-ex.py` in a folder `test-this`:
```
In [6]: test_dir = os.path.join(os.getcwd(), 'test-this/import-ex.py')

In [7]: os.path.isfile(test_dir)
Out[7]: True

In [8]: from pyomo.common.fileutils import import_file

In [9]: module = import_file(test_dir)

In [10]: module
Out[10]: <module 'import-ex' from '/Users/mmundt/Documents/devel-pyomo/mmundt-pyomo/test-this/import-ex.py'>

In [11]: 
```

Reply:
I also tried with the file `import ex.py` in `test-this`:
```
In [1]: from pyomo.common.fileutils import import_file

In [2]: import os

In [3]: test_dir = os.path.join(os.getcwd(), 'test-this/import ex.py')

In [4]: os.path.isfile(test_dir)
Out[4]: True

In [5]: module = import_file(test_dir)

In [6]: module
Out[6]: <module 'import ex' from '/Users/mmundt/Documents/devel-pyomo/mmundt-pyomo/test-this/import ex.py'>
```

Reply:
Do you want me to add these cases to the tests?